pkg/plugins/resources/toml/target.go

```diff
@@ -52,13 +52,19 @@ func (t *Toml) Target(source string, scm scm.ScmHandler, dryRun bool, resultTarg
 			}
 
 		case false:
-			queryResult, err := t.contents[i].Query(t.spec.Key)
-			if err != nil {
-				return err
+			if t.spec.AllowsMissingKey {
```

Comment:
If `AllowsMissingKey` is `true`, we need to use the multiple query method since the key access would fail on a missing key.